spec/models/base_organisation_spec.rb

```diff
@@ -56,5 +56,16 @@
       org.save
     end
   end
+  
+  describe 'friendly_id' do
+     
+     it 'should use SetupSlug service for setting slugs' do
+       service_double = class_double('SetupSlug').as_stubbed_const
```

Comment:
@tansaku: I changed what's inside brackets - it works without quotes. However it didn't work without `as_stubbed_const`.

![class_double](https://cloud.githubusercontent.com/assets/12682792/15356127/25240af6-1cf7-11e6-9211-1b43d29ef382.PNG)